index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/reset-css@5.0.2/reset.min.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>ProtoGoogle25</title>
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="header-menu__container">
+        <nav class="header-menu header-menu__parametr">
+          <a href="#" class="header-menu__link">Gmail</a>
+          <a href="#" class="header-menu__link">Images</a>
+        </nav>
+
+        <nav class="header-menu">
+          <a class="header-menu__apps-menu" href="#" title="Google apps">
+            <svg focusable="false" viewBox="0 0 24 24">
+              <path
+                d="M6,8c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM12,20c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM6,20c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM6,14c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM12,14c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM16,6c0,1.1 0.9,2 2,2s2,-0.9 2,-2 -0.9,-2 -2,-2 -2,0.9 -2,2zM12,8c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM18,14c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM18,20c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2z"
+              ></path>
+            </svg>
+          </a>
+
+          <a class="header-menu__profile-menu" href="#" title="Google account">
+            <img src="img/avatar.jpg" alt="Google avatar" />
+          </a>
+        </nav>
+      </div>
+      <!-- Здесь будет шапка сайта-->
+    </header>
+
+    <main class="main">
+      <div class="main__logo">
+        <img src="img/googlelogo_light_color_272x92dp.jpg" alt="Google" ; />
+      </div>
+
+      <div class="main__search-container">
+        <form class="search-form">
+          <div class="main__search-content-block">
+            <div class="search-content__border">
```

Comment:
у тебя нет нигде блока `search-content`, но появляется уже блок__элемент
лучше данный элемент отнести к блоку `search-input`, о котором я писал выше, так как все элементы находятся в области ответственности компонента input

```suggestion
            <div class="search-control__border">
```